docs/en/operations/settings/settings.md

```diff
@@ -109,6 +109,21 @@ Enables or disables [fsync](http://pubs.opengroup.org/onlinepubs/9699919799/func
 
 It makes sense to disable it if the server has millions of tiny tables that are constantly being created and destroyed.
 
+## function_range_max_elements_in_block {#settings-function_range_max_elements_in_block}
+
+Sets the safety threshold for data volume generated by function [range](../../sql-reference/functions/array-functions.md#range). Defines the maximum number of values generated by function per block of data (sum of array sizes for every row in a block).
+
+Possible values:
+
+-   Positive integer.
+
+Default value: `500,000,000`.
+
+**See Also**
```

Comment:
```suggestion
See also:
```

Reply:
Hi, Alexey!
In the ClickHouse style guide, we have description templates (for example [setting template] (https://github.com/ClickHouse/ClickHouse/blob/master/docs/_description_templates/template-setting.md)) so I strictly follow the agreed syntax.

Reply:
Yes, I have already acknowledged it. Thanks anyway.